restapi/src/test/java/io/stargate/web/resources/ConvertersTest.java

```diff
@@ -376,4 +404,37 @@ private static ObjectMapper testObjectMapper() {
     Server.configureObjectMapper(mapper);
     return mapper;
   }
+
+  @Test
+  public void colToValuePositiveCounter() {
+    ValueModifier modifier = Converters.colToValue("popularity", "+1", COUNTER_TABLE);
```

Comment:
Is the `+` required in value? If not, could you add a test case for an increment without an explicit `+` sign?

Reply:
Being a counter, technically yes but since we default to increment you could get away with only `1`.

Reply:
From the parsing code it looks like we simply parse it an a numeric value without enforcing `+` or `-` :thinking:

Reply:
Right we don't need to enforce anything. Input of `+1` or `1` will be an increment and `-1` will decrement.